src/main/java/org/zalando/nakadi/repository/db/SchemaRepository.java

```diff
@@ -37,12 +37,7 @@ public EventTypeSchema getSchemaVersion(final String name, final String version)
                 "WHERE ets_event_type_name = ? AND ets_schema_object ->> 'version' = ?";
 
         try {
-            final List<EventTypeSchema> schemas =
-                    jdbcTemplate.query(sql, new Object[]{name, version}, new SchemaRowMapper());
-            if (schemas.size() != 1)
-                throw new InternalNakadiException(
```

Comment:
Done in https://github.com/zalando/nakadi/commit/41ec53ac33040bd758b145dce52c1545a3353cba